### ISSUE TITLE
Restore dim-only option 

### ISSUE THREAD

**Issue by bfred-it** [sindresorhus/hide-files-on-github#41]
I was using the `dim-only` option and it was perfect, I'm not sure of why it was removed in #40 :(

Since the extension is flexible enough to allow the _hiding_ of any files, it's a great way to just bring focus to the important files, rather than completely hiding files that still affect the project, like `travis.yml`

**Comment by radiovisual** [comment]
Yeah, I never used the dimming function but the other day I ran into a situation where it would have been nice to have (I think I was trying to locate/compare a few configuration files in two different but similar repos) and it made me think that the old dimming feature would have come in handy. So I guess I vote to restore the dimming feature as well.

**Comment by bfred-it** [comment]
As an additional reason to keep files dimmed instead of hiding them is the jumping caused by hiding them. Trying to click on something that changes position is infinitely more annoying than `.travis.yml` files. This is unusable for me now :(

**Comment by radiovisual** [comment]
@bfred-it , I think you should go for the PR. I have had more use cases in these last few days where I would have liked to see this option restored. If you need some pointers, you can reference the code that was originally used to add the dim-option in commit: https://github.com/sindresorhus/hide-files-on-github/commit/fae9328aa92ed71141f4ae1577e7771dd79dcceb. Though, it won't be a `1:1` translation, since some refactors have happened since then, so if you use the old code as reference, then please reference the new code as well)

Alternatively, for anyone who wants the dimming functionality back right away, they have the option of manually installing version 1.4.1, then watch this repo for updates.

As far as the "jumping" goes, there will always be a little "jumping" because we are messing with the DOM after it has loaded. PR #42 groups the files at the top of the list, and I find that the "jump" is much better contained with that new feature.

**Comment by bfred-it** [comment]
I saw that. I'll avoid the PR because stuff would still be moving around so it'd solve nothing

**Comment by taylorjosephgriffin** [comment]
so are you not able to straight up hide files anymore? because they are only dimming for me.

**Comment by taylorjosephgriffin** [comment]
okay its hiding them but only after i refresh

**Comment by fregante** [comment]
You’re encountering https://github.com/sindresorhus/hide-files-on-github/issues/87 and #92 

Chrome doesn’t have the latest version yet
